src/node.cc

```diff
@@ -947,17 +947,56 @@ void* ArrayBufferAllocator::Allocate(size_t size) {
   return malloc(size);
 }
 
+static bool DomainHasErrorHandler(const Environment* env,
+                                  const Local<Object>& domain) {
+  HandleScope scope(env->isolate());
+
+  Local<Value> domain_event_listeners_v = domain->Get(env->events_string());
+  if (!domain_event_listeners_v->IsObject())
+    return false;
+
+  Local<Object> domain_event_listeners_o =
+      domain_event_listeners_v.As<Object>();
+
+  if (domain_event_listeners_o->IsNull())
```

Comment:
If `domain_event_listeners_v->IsObject()` then `domain_event_listeners_o->IsNull()` is always false. Or am I missing something?